components/uid-title/uidTitle.json

```diff
@@ -0,0 +1,46 @@
+{
+  "id": "uidTitle",
+  "name": "Title",
+  "type": "widget",
+  "template": "@uidTitle.tpl.html",
+  "order": "1",
+  "custom": true,
```

Comment:
This is not a custom widget: it should be 'false'.
I realize uid-text & uid-input have the same issue: could you fix this as well?